http/src/main/java/io/micronaut/http/MutableHttpResponse.java

```
@@ -185,4 +188,9 @@
     default MutableHttpResponse<B> attribute(CharSequence name, Object value) {
         return (MutableHttpResponse<B>) setAttribute(name, value);
     }
+
+    @NonNull
+    default MutableHttpResponse<B> serverPush(@NonNull URI uri, @NonNull HttpResponse<?> pushResponse) {
+        return this;
```

Should throw `UnsupportedOperationException` if not implemented